core/workspace.js

```diff
@@ -40,7 +41,7 @@ Blockly.Workspace = function(opt_options) {
   this.id = Blockly.utils.genUid();
   Blockly.Workspace.WorkspaceDB_[this.id] = this;
   /** @type {!Blockly.Options} */
-  this.options = opt_options || /** @type {!Blockly.Options} */ ({});
+  this.options = new Blockly.Options(opt_options || {});
```

Comment:
This makes sense for headless, but in rendered mode a Blockly.WorkspaceSvg instance calls it's super constructor with a options object of type ``Blockly.Options``. This would now pipe that Blockly.Options object into another Blockly.Options object.

Reply:
Good point. Looking more thoroughly through the code, changing the reference of the options object causes issues with non-headless and also I didn't realize there was a `Blockly.Options` and a `Blockly.BlocklyOptions`.

Given this, I think the earlier solution of:
`opt_options || new Blockly.Options({});`
makes more sense here.